### DISCUSSION TITLE
Add the ability to inject additionnal arguments to helm command

### DISCUSSION THREAD

**Discussion by kevin-biebuyck** [helmfile/helmfile#1039] [Ideas]
I aim to streamline the deployment of two Helm releases (A & B) using a single Helmfile. Release A generates dynamic resources like user passwords, which are required for configuring Release B. 

Currently, a script "manually" deploys Release A, extracts the values, and builds a values.yaml for Release B before deploying it. 

I plan to automate this by using a helm file in which I set up a post-install hook in Release A to generate the values.yaml for Release B, and then instruct Helmfile to pass this file when invoking Helm for Release B's deployment.

**Comment by yxxhero** [comment]
@kevin-biebuyck PR is welcome.
